ProcessLib/SmallDeformationNonlocal/SmallDeformationNonlocalFEM.h

```diff
@@ -689,71 +680,21 @@ class SmallDeformationNonlocalLocalAssembler
         std::vector<NumLib::LocalToGlobalIndexMap const*> const& /*dof_table*/,
         std::vector<double>& cache) const override
     {
-        auto const kelvin_vector_size =
-            MathLib::KelvinVector::KelvinVectorDimensions<
-                DisplacementDim>::value;
-        auto const num_intpts = _ip_data.size();
-
-        cache.clear();
-        auto cache_mat = MathLib::createZeroedMatrix<Eigen::Matrix<
-            double, kelvin_vector_size, Eigen::Dynamic, Eigen::RowMajor>>(
-            cache, kelvin_vector_size, num_intpts);
-
-        for (unsigned ip = 0; ip < num_intpts; ++ip)
-        {
-            auto const& eps = _ip_data[ip].eps;
-            cache_mat.col(ip) =
-                MathLib::KelvinVector::kelvinVectorToSymmetricTensor(eps);
-        }
-
-        return cache;
+        return ProcessLib::getIntPtEpsilon<IP_Data, DisplacementDim>(_ip_data,
+                                                                     cache);
     }
 
     std::size_t setSigma(double const* values)
     {
-        auto const kelvin_vector_size =
-            MathLib::KelvinVector::KelvinVectorDimensions<
-                DisplacementDim>::value;
-        auto const n_integration_points = _ip_data.size();
-
-        auto sigma_values =
-            Eigen::Map<Eigen::Matrix<double, kelvin_vector_size, Eigen::Dynamic,
-                                     Eigen::ColMajor> const>(
-                values, kelvin_vector_size, n_integration_points);
-
-        for (unsigned ip = 0; ip < n_integration_points; ++ip)
-        {
-            _ip_data[ip].sigma =
-                MathLib::KelvinVector::symmetricTensorToKelvinVector(
-                    sigma_values.col(ip));
-        }
-
-        return n_integration_points;
+        return ProcessLib::setSigma<IP_Data, DisplacementDim>(values, _ip_data);
```

Comment:
No underscores in type names.
https://google.github.io/styleguide/cppguide.html#Type_Names

Reply:
Changed.